### PR TITLE
added time convertion utilities

### PR DESCRIPTION
**Pull request by nhz2**
#1 Added some utility functions to convert between matlab datetime objects and the time used on the flight computer. In matlab, datetime objects take care of leap seconds and have ns precision if TimeZone is 'UTCLeapSeconds'.

### CODE REVIEW

File: MATLAB/config.m [hunk 1]
```diff
@@ -1,6 +1,7 @@
 function config
 
-global truth const
+global truth 
+global const
```
Comment (kylekrol) [inline MATLAB/config.m:4]: Is it actually a problem if the globals aren't declared on separate lines?

File: MATLAB/environmental_models/helper_functions/wrldmagm_noerror.m [hunk 2]
```diff
@@ -0,0 +1,249 @@
+function [XYZ, H, DEC, DIP, F] = wrldmagm_noerror(height, lat, lon, dyear, varargin )
+%  WRLDMAGM Use World Magnetic Model.
+%   [XYZ, H, DEC, DIP, F] = WRLDMAGM(HEIGHT, LAT, LON, DYEAR ) calculates the 
+%   Earth's magnetic field at a specific location and time using the World 
+%   Magnetic Model (WMM).  The default WMM is WMM-2015 which is valid from
+%   January 1, 2015 until December 31, 2019.
```
Comment (kylekrol) [inline MATLAB/environmental_models/helper_functions/wrldmagm_noerror.m:6]: Are we going to be updating this in the future to work in 2020+?

File: MATLAB/utl/datetime2time.m [hunk 3]
```diff
@@ -0,0 +1,8 @@
+function time = datetime2time(in_datetime,init_GPS_week_number)
```
Comment (kylekrol) [inline MATLAB/utl/datetime2time.m]: Don't want to be super picky, but could we prefix all of the utility functions with`utl_`and maybe environmental models with `env_`? I was just following something like this to make it easier to read once all of the directories get included. Also could help avoid naming collisions.
If this is a bunch of work don't worry about it tho.

**Review by kylekrol — COMMENTED** [review]
Okay, couple things really quick:
 * Do you want to substitute in the gravity model into the orbit propagator in truth_update so the sim runs off of that?
 * If I understand the new added functions correctly, we have a way to track time? Would you be comfortable adjusting what we have currently (the ns mission time counter) or is the current time tracking system sufficient?
 * Couple other comments down there in the code.

File: MATLAB/config.m (hunk 1 above)
Comment (nhz2) [reply MATLAB/config.m:4]: No I don't think so.

File: MATLAB/environmental_models/helper_functions/wrldmagm_noerror.m (hunk 2 above)
Comment (nhz2) [reply MATLAB/environmental_models/helper_functions/wrldmagm_noerror.m:6]: Yes, WMM-2020 will be released December 10, 2019. I'm not sure how long Matlab will take to update their stuff though. For now I just removed the error for inputing dates 2020+, on line 93.

**Review by stewartaslan — APPROVED** [review]
Still need to increase performance by reducing unnecessary rotations and interpolations. It probably doesn't make sense to do any linear interpolation with ODE2.

**Pull request merged**